.env.example

```diff
@@ -0,0 +1,4 @@
+NODE_ENV=development
+PORT=
+#MongoDB connection information
+MONGO_URI = 
```

Comment:
super small nit but a .env exmaple should have hard values as an example https://github.com/laravel/laravel/blob/master/.env.example